web/src/components/studies/CreateStudyForm.tsx

```diff
@@ -0,0 +1,371 @@
+
+  return (
+    <Dialog setDialogOpen={setCreateStudyFormOpen} className={styles["study-dialog"]} cy="create-study-form">
+      <h2>Create Study</h2>
+      <div className={styles["step-progress"]}>
+        <div
+          className={`${styles["step-dot"]} ${currentStep === 1 ? styles["active"] : ""} ${isValid ? styles.valid : ""}`}
+        ></div>
+        <div className={`${styles["step-dot"]} ${currentStep === 2 ? styles["active"] : ""}`}></div>
+        <div className={`${styles["step-dot"]} ${currentStep === 3 ? styles["active"] : ""}`}></div>
+      </div>
+
+      <form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
+        {/* first step */}
+        <fieldset className={getFieldsetClass(1)}>
+          <legend>Study Name</legend>
+          <Label htmlFor="shortStudyName">
+            Short Study Name*:
+            <Controller
+              name="shortStudyName"
+              control={control}
+              rules={{
+                required: "This field is required",
+                maxLength: {
+                  value: 55,
+                  message: "Maximum 55 characters",
+                },
+                pattern: {
+                  value: /^[a-zA-Z0-9 ]+$/,
+                  message: "Do not include any special characters",
+                },
+              }}
+              render={({ field }) => <Input {...field} type="text" id="shortStudyName" />}
+            />
+            <HelperText>Maximum 55 characters, do not include any special characters</HelperText>
+            {errors.shortStudyName && (
+              <Alert type="error">
+                <AlertMessage>{errors.shortStudyName.message}</AlertMessage>
+              </Alert>
+            )}
+          </Label>
+
+          <Label htmlFor="longStudyName">
+            Long Study Name:
+            <Textarea id="longStudyName" {...register("longStudyName", { maxLength: 255 })} />
+          </Label>
+        </fieldset>
+
+        <fieldset className={getFieldsetClass(1)}>
+          <legend>Ownership</legend>
+          <Label htmlFor="owner">
+            Study Owner (PI):
+            <Input
+              type="email"
+              id="owner"
+              {...register("owner")}
+              readOnly={true}
+              value={username}
+              inputClassName={styles.readonly}
+            />
+            <HelperText>
+              If you are not the study owner, contact the owner and ask them to fill out this form on their account.
+            </HelperText>
+          </Label>
+
+          <Label htmlFor="admin">
+            Study Administrator:
+            <Input type="email" id="admin" {...register("admin")} />
+            <HelperText>
+              {" "}
+              <strong>Must</strong> be a full UCL staff member, not honorary or affiliated
+            </HelperText>
+          </Label>
+
+          <Label htmlFor="controller">
+            Data Controller (organisation)*:
```

Comment:
I think this wants to be a drop down for 'UCL' or 'Other' (then input value), perhaps with multiple values. @finleybacon what do you think?